### ISSUE TITLE
Summary statistics in/from the Cell Census

### ISSUE THREAD

**Issue by pablo-gar** [chanzuckerberg/cellxgene-census#22]
TLDR -- There may a be a need to create summary statistics using the cell census. This discussion centers around:

- What are useful statistics/summary tables?
   - marker genes
   - average gene expression per cell type 
- Should these be stored in the Cell Census SOMA object, or should they be provided on-demand via an API?

Ambrose Carr

> @[angela.pisco@czbiohub.org](mailto:angela.pisco@czbiohub.org) proposes to review this table and flag additional summary statistics that would help her evaluate the census as a part of her annotation work.

Bruce Martin

> What criteria do we apply to inclusion in this table?  Many of these are trivial to compute in a notebook (and can be done very quickly if they only rely on the obs dataframe).  At some point, it seems like there are diminishing returns to maintaining a giant list of summary statistics which demo notebooks would better serve.

I think we would benefit from some first principles/guidelines...

Ambrose Carr in reference to "Cell Census summary cell counts"  – `census_obj[“census_info”][“summary_cell_counts”]` – `SOMADataframe`

> Important point. 
> 
> In addition to being able to access the full census source data (this document), I want our users to be able to access a carefully curated set of summary statistics that provide access to information that describe cell types. 
> 
> @[pgarcia-nieto@chanzuckerberg.com](mailto:pgarcia-nieto@chanzuckerberg.com) pointed out that much of this information is already available via the census. 
> 
> But perhaps this table is not the right place for it?
> 
> I'd be happy to get on a call and nail down some principles.

Bruce Martin

> You might consider defining the statistics and the means of publishing the stats separately. I could see utility in having the stats in multiple places, eg., in an HTML page for humans, in the census via API for notebooks, etc.

Angela Pisco

> I agree that most of the statistics can rather be done in a notebook and therefore should not be added to the schema

**Comment by atolopko-czi** [comment]
Perhaps useful summarization methods could be provided via normal Python modules rather tha notebooks, if only to allow easier invocation (avoid copy & paste).

**Comment by brianraymor** [comment]
@pablo-gar to refine and decompose into actionable prioritized issues.

**Comment by pablo-gar** [comment]
Based on the discussion it is preferable to either:

1. Provide notebooks with calculations explained.
2. Add functions to the new experimental package.
3. And for very simple cases add new cell/gene metadata variables to the Census or dataset schema (see for example #289
